client/gutenberg/extensions/map/settings.js

```diff
@@ -0,0 +1,96 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
```

Comment:
Since no deps listed here, let's remove the _"Internal dependencies"_ comment. 💪

Reply:
Removed this, and a couple of other headers for nothing here: https://github.com/Automattic/wp-calypso/pull/28498/commits/35139cbf6b7f595088de3146fcb622f9f110dee9